src/SFA.DAS.Forecasting.Application/Infrastructure/Registries/PaymentsRegistry.cs

```diff
@@ -1,50 +1,22 @@
-﻿using System.Net.Http;
-using SFA.DAS.Forecasting.Application.Infrastructure.Configuration;
+﻿using SFA.DAS.Forecasting.Application.Infrastructure.Configuration;
 using SFA.DAS.Forecasting.Application.Payments.Services;
-using SFA.DAS.Http;
-using SFA.DAS.Http.TokenGenerators;
-using SFA.DAS.NLog.Logger.Web.MessageHandlers;
 using SFA.DAS.Provider.Events.Api.Client;
 using SFA.DAS.Provider.Events.Api.Client.Configuration;
-using StructureMap;
-using PaymentsEventsApiConfiguration = SFA.DAS.Forecasting.Application.Infrastructure.Configuration.PaymentsEventsApiConfiguration;
+using SFA.DAS.Provider.Events.Api.Client.DependencyResolution;
 
 namespace SFA.DAS.Forecasting.Application.Infrastructure.Registries
 {
-    public class PaymentsRegistry : Registry
+    public class PaymentsRegistry : PaymentsEventsApiClientRegistry
     {
         public PaymentsRegistry()
         {
-            // TODO: Bring back when AAD has been fully implemented.
-            //For<PaymentsEventsApiConfiguration>().Use(c => c.GetInstance<ForecastingConfiguration>().PaymentsEventsApi).Singleton();
-            //For<IPaymentsEventsApiConfiguration>().Use(c => c.GetInstance<PaymentsEventsApiConfiguration>());
+            For<PaymentsEventsApiClientConfiguration>().Use(c => c.GetInstance<ForecastingConfiguration>().PaymentsEventsApi).Singleton();
+            For<IPaymentsEventsApiClientConfiguration>().Use(c => c.GetInstance<PaymentsEventsApiClientConfiguration>());
 
             if (ConfigurationHelper.IsDevEnvironment)
             {
                 For<IPaymentsEventsApiClient>().Use<DevPaymentsEventsApiClient>();
             }
-            else
-            {
-                For<IPaymentsEventsApiClient>().Use<PaymentsEventsApiClient>().Ctor<HttpClient>("httpClient").Is(c => CreateClient(c));
```

Comment:
where are we registering the actual PaymentsEventsApiClient?

Reply:
It's in the base class of the registry

Reply:
should the dev registration calls 
`For<IPaymentsEventsApiClient>().ClearAll().Use<PaymentsEventsApiClient>().Ctor<HttpClient>("httpClient").Is(c => CreateClient(c));`

Reply:
Fixed with ClearAll